x-pack/plugins/security_solution/common/endpoint/types/actions.ts

```diff
@@ -14,11 +14,15 @@ import {
 
 export type ISOLATION_ACTIONS = 'isolate' | 'unisolate';
 
-export type ResponseActions =
-  | ISOLATION_ACTIONS
-  | 'kill-process'
-  | 'suspend-process'
-  | 'running-processes';
+export const RESPONSE_ACTION_COMMANDS = [
```

Comment:
can you please move this out of the `types` file and into perhaps `common/endpoint/constants`.

Also - I seem to perhaps remember you also creating something like this on the server/API, no? if so, any chance we can use a single source?

Reply:
Ohhh. I see. I think you defined it here because you are inferring a type from the content of the array below on line 25. Is that right?